web/yo/app/scripts/directives/reviewpanel.js

```diff
@@ -377,8 +377,8 @@ angular.module('oncokbApp')
                                     rejectItem(items);
                                 }
                             } else {
-                                if(oncogenicChange) uuid = oncogenicUUID;
```

Comment:
If you look at line 288 inside the rejectItem function, uuid is the uuid collaborative string object, instead of the real string value.